tests/unit/test_ruleset.py

```diff
@@ -24,7 +24,7 @@
 
 def test_ruleset():
     tests_dir = os.path.dirname(os.path.dirname(__file__))
-    lol_ruleset_path = os.path.join(tests_dir, "data", "lol-ruleset.json")
```

Comment:
What about leaving the test for `json` here as well? So we can be sure, we are not breaking the current rulesets...

Reply:
Yeah but the only error we could face here is parsing error by the `PyYAML` library. But `YAML` by definition is a superset of `JSON`, so we could also leave it that way. But also, I guess when we are testing, it's always better to consider every cases. Let me know what you think.

Reply:
I agree with the JSON compatibility, but I am thinking more about the structure. (Some JSON file is YAML compatible, but has e.g. some attributes in the wrong place. It is just a theoretical case, but for sure.)

Reply:
I see. Yeah of course bugs could arise in cases where you least expect it. I'll handle both of the versions of `lol-ruleset`.